components/eamxx/src/physics/dp/impl/dp_apply_iop_forcing_impl.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef DP_APPLY_IOP_FORCING_IMPL_HPP
+#define DP_APPLY_IOP_FORCING_IMPL_HPP
+
+#include "dp_functions.hpp" // for ETI only but harmless for GPU
+
+namespace scream {
+namespace dp {
+
+/*
+ * Implementation of dp apply_iop_forcing. Clients should NOT
+ * #include this file, but include dp_functions.hpp instead.
+ */
+
+template<typename S, typename D>
+KOKKOS_FUNCTION
+void Functions<S,D>::apply_iop_forcing(const Int& nelemd, const uview_1d<element_t>& elem, hvcoord_t& hvcoord, const hybrid_t& hybrid, const timelevel_t& tl, const Int& n, const bool& t_before_advance, const Int& nets, const Int& nete)
```

Comment:
In F90, this function (and some others) live in the `eam/src/dynamics/` folder with calls to homme. Not sure exactly how the hooks to homme work in eamxx, but will this need to live in the `eamxx/src/dynamics` subdirectory instead of dp?

Reply:
Yeah, I expect some of this stuff will need to be moved but I'm not worried aboutt that right now.